src/analysis/retail/druid/restoration/CombatLogParser.ts

```diff
@@ -125,6 +126,9 @@ class CombatLogParser extends CoreCombatLogParser {
     manaTracker: ManaTracker,
     hpmDetails: RestoDruidHealingEfficiencyDetails,
     hpmTracker: HealingEfficiencyTracker,
+
+    //Tier
+    Dragonflight2pc: Dragonflight2pc,
```

Comment:
To keep it in line with the rest of the dependencies:
```suggestion
    dragonflight2pc: Dragonflight2pc,
```